package.json

```diff
@@ -24,9 +24,9 @@
 		"publish": "electron-forge publish",
 		"lint": "eslint --ext .ts,.tsx,.js,.jsx,.mjs .",
 		"format": "prettier . --write",
-		"test": "jest",
-		"test:watch": "jest --watch",
-		"e2e": "npx playwright install && npx playwright test",
+		"test": "NODE_ENV=test jest",
+		"test:watch": "NODE_ENV=test jest --watch",
+		"e2e": "NODE_ENV=test npx playwright install && npx playwright test",
```

Comment:
From my testing locally, this does not achieve the desired outcome. 

<details><summary>Testing diff</summary>

```diff
diff --git a/e2e/e2e-helpers.ts b/e2e/e2e-helpers.ts
index feb764d..1e404eb 100644
--- a/e2e/e2e-helpers.ts
+++ b/e2e/e2e-helpers.ts
@@ -4,6 +4,7 @@ import { _electron as electron } from 'playwright';
 export async function launchApp( testEnv: NodeJS.ProcessEnv = {} ) {
 	// find the latest build in the out directory
 	const latestBuild = findLatestBuild();
+	console.log( '>>>', process.env.NODE_ENV );
 
 	// parse the packaged Electron app and find paths and other info
 	const appInfo = parseElectronApp( latestBuild );
diff --git a/src/index.ts b/src/index.ts
index dc7a605..33d9084 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -6,6 +6,7 @@ import {
 	type IpcMainInvokeEvent,
 	globalShortcut,
 	Menu,
+	dialog,
 } from 'electron';
 import path from 'path';
 import * as Sentry from '@sentry/electron/main';
@@ -26,6 +27,12 @@ import setupWPServerFiles from './setup-wp-server-files';
 import { setupUpdates } from './updates';
 import { stopAllServersOnQuit } from './site-server'; // eslint-disable-line import/order
 
+app.whenReady().then( () => {
+	dialog.showMessageBox( {
+		message: `ENV --- ${ process.env.NODE_ENV } ---`,
+	} );
+} );
+
 Sentry.init( {
 	dsn: 'https://97693275b2716fb95048c6d12f4318cf@o248881.ingest.sentry.io/4506612776501248',
 	debug: true,

```

</details>

I believe we need to set `NODE_ENV` for the build itself. Currently, the [latest build](https://github.com/Automattic/studio/blob/e58a267a9c0463a4f80c65893c577b190403c17f/e2e/e2e-helpers.ts#L5-L6) existing in the `out` directory is used. We may need an explicit script for building E2E test builds with the proper `NODE_ENV` set. This would need to be run _prior_ to running the E2E test suite, otherwise if a production build happened to be present in the `out` directory, it would be used.

Reply:
I see your point. It does seem like it may be a bit convoluted to create a new script just to leverage `NODE_ENV`. Instead, it does seem that we could use [process.env.E2E](https://github.com/Automattic/studio/blob/e58a267a9c0463a4f80c65893c577b190403c17f/e2e/e2e-helpers.ts#L16), which is already being set and returns true when the E2E tests run.

I've updated the Sentry init config to utilize `process.env.E2E` as well. What do you think of this approach?